study-builder/studies/mpc/study-pdfs.conf

```diff
@@ -171,6 +171,167 @@
           ]
         }
       ]
+    },
+    {
+      "name": "mpcproject-followup",
+      "filename": "mpcproject-followup",
+      "displayName": "MPC followup pdf"
+      "versions": [
+        {
+          "tag": "v1",
+          "start": "2020-08-01T00:00:00Z",
+          "end": null,
+          "sources": [
+            {
+              "type": "PARTICIPANT"
+            },
+            {
+              "type": "ACTIVITY",
+              "activityCode": "FOLLOWUP",
+              "versionTag": "v1"
+            }
+          ],
+          "files": [
+            {
+              "filepath": "pdfs/followup_consent_text.pdf",
+              "type": "CUSTOM",
+              "substitutions": [],
+              "fields": {}
+            },
+            {
+              "filepath": "pdfs/followup_contact.pdf",
+              "type": "MAILING_ADDRESS",
+              "fields": {
+                "firstName": "firstName",
+                "lastName": "lastName",
+                "street": "street",
+                "city": "city",
+                "state": "state",
+                "zip": "zip",
+                "country": "country",
+                "phone": "phone"
+              }
+            },
+            {
+              "filepath": "pdfs/followup_curr_medication.pdf",
```

Comment:
I couldn't add a comment to the PDF files on github, but wanted to ask: these and the "details" PDF files already has `FALSE` and `NO` text in the file itself, will that cause any issues?

Reply:
After some back and forth  with Mary and Lauren I had a default value for one question (FALSE) . 
Since we implemented these Yess/No (True/False) questions as Picklists , Mary wanted value if not checked.